tests/test_accessibility.py

```diff
@@ -0,0 +1,156 @@
+"""
```

Comment:
Might be worth using pytest.mark to mark accessibility, performance, initialisation, etc tests in case we ever want to just test accessibility, etc. Thoughts on this? 

Also should utils have a relative import? you should be able to do tests/utils, silly point but some people dislike relative imports.

Reply:
Relative imports replace. Following discussion custom pytest marks removed on account of lack of intellisense support and so likelihood of typos etc. being introduced